modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -3,31 +3,31 @@
     <span class="apos-notification__indicator">
       <AposIndicator
         :icon="iconComponent" class="apos-notification__indicator__icon"
-        :icon-size="icon ? 16 : 12"
+        :icon-size="notification.icon ? 16 : 12"
```

Comment:
The forest of props was becoming unmaintainable and mirrored the notification object 1-1, so I changed to a single notification prop.